File: CaseTracker/Models/Payment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CaseTracker.Models
{
	public class Payment
	{
		public int Id { get; set; }
		public string cardNumber { get; set; }
		public int expiresmonth { get; set; }
		public int expiresyear { get; set; }
		public string fname { get; set; }
		public string lname { get; set; }
	}
}

File: CaseTracker/Scripts/case/payment.js
$(document).ready(() => {
	$("#submit").click(submitUserInfo);
});

let submitUserInfo = (e) => {
	e.preventDefault();

	if (!$("#proform").valid()) {
		//alert("missing");
		return;
	}

	let cardNumber = $("input[name=cardnumber]").val();
	let expiresmonth = $("select[name=expiresmonth]").val();
	let expiresyear = $("select[name=expiresyear]").val();
	let fname = $("input[name=fname]").val();
	let lname = $("input[name=lname]").val();

	$.post("/api/Payment", {
		"cardnumber": cardNumber,
		"expiresmonth": expiresmonth,
		"expiresyear": expiresyear,
		"fname": fname,
		"lname": lname
	}, proSuccess);
}

let proSuccess = () => {
	let pathArray = window.location.pathname.split("/");
	console.log(pathArray);
	let target = "/Home/Pro";
	if (pathArray[1] === "en") {
		target = "/en" + target;
	} else if (pathArray[1] === "el") {
		target = "/el" + target;
	}
	window.location.replace(target);
}